securedpreferencestore/src/main/java/devliving/online/securedpreferencestore/EncryptionManager.java

```diff
@@ -107,8 +114,15 @@
 
     SecuredPreferenceStore.KeyStoreRecoveryNotifier mRecoveryHandler;
 
-    public EncryptionManager(Context context, SharedPreferences prefStore, SecuredPreferenceStore.KeyStoreRecoveryNotifier recoveryHandler)
```

Comment:
we need to keep the older constructor in case someone wants to keep using the default.

Reply:
Is it intended for endusers to use the EncryptionManager? I see that it is not PackagePrivate, thus indicating that end users may get a hold of it.
If I can provide a reliable upgrade mechanism, which would also address your compatibility point, do you then believe it is still necessary to keep the old constructor?

Reply:
Yes, this is intended for the end user specially in cases where the user is not using the secured preference but wants to use the library to encrypt/decrypt data. 

Yes, it's not mandatory to keep the old constructor if we have a smooth non-confusing API to upgrade to.